code/Cargo.toml

```diff
@@ -354,7 +354,7 @@ wasmi-validation = { version = "0.5", default-features = false }
 cosmwasm-vm = { git = "https://github.com/ComposableFi/cosmwasm-vm", rev = "1cc47415577d67448ff483ad530b889cceafde77", default-features = false }
 cosmwasm-vm-wasmi = { git = "https://github.com/ComposableFi/cosmwasm-vm", rev = "1cc47415577d67448ff483ad530b889cceafde77", default-features = false }
 cosmwasm-orchestrate = { git = "https://github.com/ComposableFi/cosmwasm-vm", rev = "1cc47415577d67448ff483ad530b889cceafde77" }
-serde-json-wasm = { git = "https://github.com/dzmitry-lahoda-forks/serde-json-wasm", rev = "8a7e522c0e4de36a6dfb535766f26a9941017d81", default-features = false }
+serde-json-wasm = { git = "https://github.com/CosmWasm/serde-json-wasm/", rev = "5629f4222b020e2597dc15ca29b120f82fccca3a", default-features = false, features = ["unstable"] }
```

Comment:
@mina86 they changed 128 string ecoding to u128 as 54bit floating point numbers.

is all serde using u128 as string like asset id work correctly?